packages/common/src/utils/stream-utils.ts

```diff
@@ -366,6 +367,12 @@ export class StreamUtils {
             logEntry.expirationTime
           }. Loading the stream with 'sync: SyncOptions.ALWAYS_SYNC' will restore the stream to a usable state, by discarding the invalid commits (this means losing the data from those invalid writes!)`
         )
+      } else if (logEntry.expirationTime && logEntry.expirationTime - timestamp < 12 * 60 * 60) {
+        logger.warn(
```

Comment:
since `timestamp` can come from the anchor commit time OR from "now" if the stream isn't actually anchored yet, this log message is a bit misleading, since the message implies the stream is already anchored in all cases

Reply:
`anchorTime=${logEntry.timestamp}` will be null if there is no anchor but I see what you mean about the wording.